java-frontend/src/main/java/org/sonar/java/se/ExplodedGraph.java

```diff
@@ -113,7 +117,7 @@ public Tree syntaxTree() {
     public final ProgramPoint programPoint;
     @Nullable
     public final ProgramState programState;
-    private final List<Node> parents;
+    private final Map<Node, MethodYield> parents;
```

Comment:
This implies one property : A node can only be reached from a parent via a unique method yield. I am not really sure this is true.